auto_rx/autorx/scan.py

```diff
@@ -969,7 +969,7 @@ def sonde_search(self, first_only=False):
             (_freq_decimate, _power_decimate) = peak_decimation(freq / 1e6, power, 10)
             scan_result["freq"] = list(_freq_decimate)
             scan_result["power"] = list(_power_decimate)
-            scan_result["timestamp"] = datetime.datetime.utcnow().isoformat()
+            scan_result["timestamp"] = datetime.datetime.now(datetime.timezone.utc).isoformat()
```

Comment:
I noticed that this is going to change behaviour:
```
>>> datetime.datetime.utcnow().isoformat()
'2024-03-27T22:36:59.792134'
>>> datetime.datetime.now(datetime.timezone.utc).isoformat()
'2024-03-27T22:37:09.889948+00:00'
```
Maybe it should get a more specific time format?